### PR TITLE
RT-1.7,RT-1.5,RT-1.10 | Add BGP route-policy config under AFI/SAFI by default

### PR DESCRIPTION
**Pull request by arvbaska1**
Made below changes to the testcases:

- Added BGP route-policy config by default without the deviation.Per RFC8212 for eBGP a route-policy is a must to allow routes to be advertised/received.

- Route-policy is configured under the AFI/SAFI under the neighbor.

- Removed RoutePolicyUnderNeighborAfiSafi flag from deviations.go

### CODE REVIEW

**Comment by xw-g** [comment]
@arvbaska1, probably we don't need to remove the deviation as it might be used by others.

**Comment by arvbaska1** [comment]
> 

Okay, so keep this deviation flag in the deviations.go if used by anyone else ?

File: feature/bgp/prefixlimit/ate_tests/bgp_prefix_limit_test/bgp_prefix_limit_test.go [hunk 1]
```diff
@@ -262,15 +262,9 @@ func createBGPNeighbor(localAs, peerAs, pLimit uint32, restartTime uint16) *oc.N
...
-			if *deviations.RoutePolicyUnderNeighborAfiSafi {
-				rpl := afisafi.GetOrCreateApplyPolicy()
-				rpl.ImportPolicy = []string{rplName}
-				rpl.ExportPolicy = []string{rplName}
-			} else {
-				rpl := nv4.GetOrCreateApplyPolicy()
-				rpl.ImportPolicy = []string{rplName}
-				rpl.ExportPolicy = []string{rplName}
-			}
+			rpl := afisafi.GetOrCreateApplyPolicy()
```
Comment (guoshiuan) [inline feature/bgp/prefixlimit/ate_tests/bgp_prefix_limit_test/bgp_prefix_limit_test.go:265]: This PR breaks the test status of some vendors.

**Comment by xw-g** [comment]
Can you help update the README to reflect your changes too? (especially the AFI-SAFI part). Thanks.

**Comment by arvbaska1** [comment]
> Can you help update the README to reflect your changes too? (especially the AFI-SAFI part). Thanks.

Updated the Readme files

**Review by guoshiuan — APPROVED** [review]

File: feature/bgp/prefixlimit/ate_tests/bgp_prefix_limit_test/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ BGP Prefix Limit
 
 ## Procedure
 
-*   Configure BGP session between ATE port-1 and DUT port-1.
+*   Configure eBGP session between ATE port-1 and DUT port-1,with an Accept-route all import-policy/export-policy under the neighbor AFI/SAFI.
```
Comment (xw-g) [inline feature/bgp/prefixlimit/ate_tests/bgp_prefix_limit_test/README.md:9]: @arvbaska1  The code says it's under peer-group, please help update this README if you agree. Same applies to other changes in this PR. Thanks. 

cc: @guoshiuan

Comment (arvbaska1) [reply feature/bgp/prefixlimit/ate_tests/bgp_prefix_limit_test/README.md:9]: Yes, will update the Readme for the tests.

**Pull request merged**